scripts/qemucommand.py

```diff
@@ -102,12 +102,8 @@ def command_line(self):
         cmdline += [
             "-serial", "tcp:127.0.0.1:%d,server,nowait" % self.serial_port,
```

Comment:
While we're at it, maybe we should remove this serial option by default in --no-gui. When not present, the system is directly usable with stdio and I don't remember having to use the serial port to debug something.
It could be made optional just in case though.

Reply:
I only didn't move it because I was afraid that it might break some tests and I also was lazy to check that. I guess I'll move it and see what happens.

Reply:
@lbonn Didn't realize that in this case kernel boot log gets into the test log, which is probably not something that we want.

Reply:
What do you mean by test logs?

Currently, there is no output at all with `--no-gui`, so I thought it would be useful to at least be able to get a prompt there. kernel logs can be annoying sure, but they are also useful when debugging u-boot / boot issues.

It could also be behind a switch though and we'd also better make sure that the message "To connect to the serial console:" only shows when it would actually work.

Reply:
@lbonn sorry, I mean oe-selftest logs

Reply:
ah ok, yes that makes sense. Anyway looks like it requires more care than I thought, so I guess we can already merge this PR and come back on that if needed.